stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/state/messages/list/MessageFocusState.kt

```diff
@@ -0,0 +1,16 @@
+package io.getstream.chat.android.compose.state.messages.list
+
+/**
+ * Represents the message focus state, in case the user jumps to a message.
+ */
+public sealed class MessageFocusState
+
+/**
+ * Represents the state when the message is currently being focused.
+ */
+public object MessageFocused : MessageFocusState()
+
+/**
+ * Represents the state when we've removed the focus from the message.
+ */
+public object FocusRemoved : MessageFocusState()
```

Comment:
This is just a suggestion, I think if we have the same prefix (Message-) would be better to maintain consistency. 😄